src/auth/login.test.ts

```diff
@@ -67,6 +67,14 @@ const testSamUserAllowances = {
   details: testSamUserAllowancesDetails,
 };
 
+const mockSamUserCombinedState: SamUserCombinedStateResponse = {
+  samUser: mockSamUserResponse,
+  terraUserAllowances: testSamUserAllowances,
+  terraUserAttributes: { marketingConsent: false } as SamUserAttributes,
```

Comment:
Unnecessary cast can be removed.

```suggestion
  terraUserAttributes: { marketingConsent: false },
```